functions/socless_log_findings/lambda_function.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 Twilio, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+from socless import *
+from datetime import datetime
+import boto3
+import json
+
+def lambda_handler(event,context):
+    def handle_state(event_context, findings, bucket_name):
```

Comment:
Let bucket_name be an environment variable so we the user of the integration doesn't have to worry about it